src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -108,10 +110,10 @@ CEvaluationResult* CCrossValidation::evaluate_impl()
 		SG_DEBUG("result of cross-validation run %d is %f\n", i, results[i])
 
 		/* Emit the value*/
-		std::string obs_value_name{"cross_validation_run"};
-		ObservedValue cv_data{i, obs_value_name, make_any(storage),
-		                      CROSSVALIDATION};
-		observe(cv_data);
+		observe(
+		    ObservedValue::make_observation<CrossValidationStorage*>(
```

Comment:
What about renaming make_observation to observe (shorter, neater)

Reply:
I would prefer to leave it as it is, since it may cause confusion if we changed it to `observe`. Namely, `observe` emit a value, `make_observation` is a utility that creates an observable object.

Reply:
Good point :)